pkg/client/client_test.go

```diff
@@ -312,7 +312,7 @@ var _ = Describe("ConmonClient", func() {
 				rssAfter := vmRSSGivenPID(pid)
 				GinkgoWriter.Printf("VmRSS after: %d\n", rssAfter)
 				GinkgoWriter.Printf("VmRSS diff: %d\n", rssAfter-rssBefore)
-				Expect(rssAfter - rssBefore).To(BeNumerically("<", 1200))
+				Expect(rssAfter - rssBefore).To(BeNumerically("<", 1500))
```

Comment:
The integration tests seemed to flake more lately. Not really related to this PR, but should make them more resilient.